pkg/apis/openebs.io/upgrade/v1alpha1/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Please refer to https://github.com/openebs/maya/tree/master/pkg/apis/openebs.io/catalog/v1alpha1
for doc.go, register.go & other xyz_file.go